zap/src/test/resources/org/zaproxy/zap/spider/parser/html/AppletElementsSpiderHtmlParser.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+<base href="http://example.com/sample/">
+<meta charset="UTF-8">
+<title>Input Elements - Spider HTML Parser</title>
+</head>
+<body>
+<!--  Ignored: -->
+<applet></applet>
+
+<!--  src -->
+<applet type="deprecated" src="/cool/applet/file.class"  alt="badpractice" code="somecode" height="320" width="480" >
```

Comment:
Irrelevant attributes (and elements) should be removed.

Reply:
I was adding them because my linter was complaining about missing required attributes for the tag. 
I will remove them wherever I added them for simplicity

Reply:
It probably wont' matter after you tackle the other comment but there's a dbl space leading the alt attr.

If we were actually building web content that'd be fine (to include what the linter wanted) but for test purposes it's best to just focus on what's needed for the test.

Reply:
Align, height, width, alt, etc can all be dropped throughout the doc right?

Reply:
> Align, height, width, alt, etc can all be dropped throughout the doc right?

Forgot to push, the code was there :)

Reply:
The `type` in the `input` elements should be removed.